executor/scripts/test_local_executor.py

```
@@ -0,0 +1,256 @@
+                    print("  ✗ Local executor namespace not initialized")
+                    return {"connected": False, "data": data}
+            else:
+                print(f"  ✗ Status check returned {response.status_code}")
+                return {"connected": False, "error": response.text}
+        except Exception as e:
+            print(f"  ✗ Error checking executor status: {e}")
+            return {"connected": False, "error": str(e)}
+
+
+async def dispatch_test_task(
+    backend_url: str, prompt: str, anthropic_api_key: str = None
```

Ruff (0.14.14)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)